CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ option(WITH_OFFLINE "Build offline interface?" OFF)
 option(USE_DEPTH_COMPUTE_STUBS "Use empty generated libs?" OFF)
 option(USE_DEPTH_COMPUTE_OPENSOURCE "Use an open source implementation?" OFF)
 option(WITH_GLOG_DEPENDENCY "Build with GLOG dependency to be used for logging" ON)
```

Comment:
There is a readme where we explain in more detail these options. Add these new options as well.

Reply:
Updated.